docs/UML diagrams/Ingredients.puml

```diff
@@ -0,0 +1,32 @@
+@startuml
+'https://plantuml.com/class-diagram
+
+IngredientList <-- MealCompanionSession
+Ingredient "*" <-- IngredientList
+
+class Ingredient {
+-quantity: double
+-name: String
++getQuantity()
++setQuantity(double)
++getName()
+}
+
+class IngredientList {
+
++size(): int
++remove(int)
++get(int): Ingredient
++getIngredients() : IngredientList
+}
+
+Class MealCompanionSession {
+- controlFLow: MealCompanionControlFlow
+- recipes: RecipeList
+- ui: MealCompanionUI
+- ingredientStorage: IngredientStorage
+
+}
+
+
```

Comment:
<img width="825" alt="Screen Shot 2023-03-31 at 11 26 08 AM" src="https://user-images.githubusercontent.com/88696064/229015255-32d0c9d8-908f-476d-929d-b6636d6f22ba.png">

The "+" and "-" are not showing correctly here.